spec/lib/whitehall_importer/create_edition_spec.rb

```diff
@@ -292,5 +275,69 @@
         described_class.call(document: document, whitehall_edition: whitehall_edition)
       }.to raise_error(WhitehallImporter::AbortImportError)
     end
+
+    context "when a draft document was force scheduled" do
```

Comment:
This feels like quite a lot of overhead to be adding when the only difference between the two contexts is the value of the second revision history event. Hopefully more liberal use of RSpec `let` blocks as suggested earlier will provide a path for making this more concise.